### PR TITLE
[Functions] (fix): Functions cost estimation correctly uses Registry fee

### PR DESCRIPTION
**Pull request by justinkaseman**
- Corrects issue where cost estimation was not using registry fee

### CODE REVIEW

**Review by KuphJr — APPROVED** [review]

File: contracts/src/v0.8/dev/functions/FunctionsOracle.sol [hunk 1]
```diff
@@ -165,7 +165,7 @@ contract FunctionsOracle is
       gasPrice
     );
     uint96 requiredFee = getRequiredFee(data, billing);
```
Comment (bolekk) [inline contracts/src/v0.8/dev/functions/FunctionsOracle.sol:167]: Can we call this oracleFee or donFee?

**Review by bolekk — COMMENTED** [review]
I see that FunctionsOracleMigration.sol has the same bug. Do you want to fix it there as well? Or maybe remove those *Migration files in a separate PR if they're not needed?

Also is it easy to add a unit test?

**Review by pinebit — APPROVED** [review]

File: contracts/src/v0.8/dev/functions/FunctionsOracle.sol (hunk 1 above)
Comment (justinkaseman) [reply contracts/src/v0.8/dev/functions/FunctionsOracle.sol]: Done. Good suggestion, it's a lot more clear.

**Comment by justinkaseman** [comment]
> I see that FunctionsOracleMigration.sol has the same bug. Do you want to fix it there as well? Or maybe remove those *Migration files in a separate PR if they're not needed?
> 
> Also is it easy to add a unit test?

I'm going to leave the migration files alone in this PR since they are currently unused, and they will also need their own PR soon once `hardhat-upgrades` comes out with their new version. Once that is out I am going to re-add the unit test for migrations.

As for unit testing this change, it's not the easiest because the fee is hard coded to 0. I'll add it once we have the fees hooked up to config.

**Review by bolekk — APPROVED** [review]

**Pull request merged**